runtime/afl.c

```diff
@@ -15,6 +15,11 @@
 /* Runtime support for afl-fuzz */
 #include "caml/config.h"
 
+/* Values used by the instrumentation logic (see cmmgen.ml) */
+static unsigned char afl_area_initial[1 << 16];
```

Comment:
So, every OCaml program is 64 Kbytes bigger, just in case AFL is applied to it?

I know this is orthogonal to the issue at hand, but I'd really like to see on-demand allocation here.

Reply:
Ah, it wasn't just me bothered by this! I'll open an issue to track it...